_shared/project/tests/pytest_plugins/cookiecutter.py

```diff
@@ -0,0 +1,48 @@
+{% if cookiecutter.get("postgres") == "yes" and cookiecutter.get("_directory") in ["pyapp", "pyramid-app"]  %}
+from os import environ
+
+{% endif %}
+import factory.random
+import pytest
+{% if cookiecutter.get("postgres") == "yes" and cookiecutter.get("_directory") in ["pyapp", "pyramid-app"]  %}
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from {{ cookiecutter.package_name }}.db import Base
+{% endif %}
+
+
+{% if cookiecutter.get("postgres") == "yes" and cookiecutter.get("_directory") in ["pyapp", "pyramid-app"]  %}
+@pytest.fixture(scope="session")
+def db_engine():
+    engine = create_engine(environ["DATABASE_URL"])
+
+    Base.metadata.drop_all(engine)
+    Base.metadata.create_all(engine)
+
+    return engine
```

Comment:
Delete all DB tables and recreate them at the begin of each unittest _session_ (not inbetween each test). This is what our other apps do, it means the tests are always run against a DB schema that's up to date with the current state of the model code.